packages/manager/modules/nutanix/src/routing.js

```diff
@@ -61,10 +62,10 @@ export default /* @ngInject */ ($stateProvider) => {
             property: 'targetSpec.nodes',
           },
           {
-            title: $translate.instant('nutanix_cluster_list_licence'),
-            property: 'targetSpec.license',
+            title: CLUSTER_LIST_TITLE.TYPE_OF_PACK,
```

Comment:
I think that for the user it is better to keep something translated. Because `Type of pack` can easily be translated.

Reply:
Hi @ghyenne, we don't have to translate `Type of pack` that's why we kept it under constant. Its going to be same in all the languages.